proto/celestia/core/v1/da/data_availability_header.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package celestia.core.v1.da;
 
-option go_package = "github.com/celestiaorg/celestia-app/proto/celestia/core/v1/da";
+option go_package = "github.com/celestiaorg/celestia-app/v2/proto/celestia/core/v1/da";
```

Comment:
[no change needed] is there a way to sanity check that that this name is correct? I guess I'm asking if any of the `proto` CI checks would fail if this line pointed to an incorrect Go package